plugins/engines/ospray/ispc/render/SimulationRenderer.ispc

```diff
@@ -505,7 +505,7 @@ inline vec3f SimulationRenderer_shadeRay(
 
             if (attributes.opacity > 0.01f && moreRebounds)
             {
-                // Compute indirect lightening contribution
+                // Compute indirect lightning contribution
```

Comment:
it's even 'lighting'. No thunderstorms here :)

Reply:
oops. yea..so keep this `lightening` right ?

Reply:
hehe...yea..it is `lighting`! Ok, fixed